dhctl/pkg/kubernetes/actions/converge/converge.go

```diff
@@ -315,6 +432,84 @@ func (r *Runner) converge() error {
 	return nil
 }
 
+func (r *Runner) replaceKubeClient(ctx context.Context, state map[string][]byte) (err error) {
+	tmpDir := filepath.Join(app.CacheDir, "converge")
+
+	err = os.MkdirAll(tmpDir, 0o755)
+	if err != nil {
+		return fmt.Errorf("failed to create cache directory for NodeUser: %w", err)
+	}
+
+	privateKeyPath := filepath.Join(tmpDir, "id_rsa")
```

Comment:
for prevent misunderstanding, let rename to id_rsa_converger

Reply:
Fixed